habitat-lab/habitat/utils/visualizations/maps.py

```diff
@@ -7,7 +7,7 @@
 import os
 from typing import Any, Dict, List, Optional, Sequence, Tuple
 
-import imageio
+import imageio.v2 as imageio
```

Comment:
why do we want to upgrade imageio version to v2?

Reply:
There are some warnings when running the code, forgot to revert.